Snowdog_Components/components/Molecules/table/_table-variables.scss

```diff
@@ -3,7 +3,9 @@ $table__width                                                : 100% !default;
 $table__padding                                              : 0 !default;
 $table__background                                           : $white !default;
 $table__margin                                               : $spacer 0 !default;
-$table__border                                               : $border-width-base $border-style-base $gray-light !default;
+$table__border-width-base                                    : $border-width-base;
```

Comment:
please add `!default` https://thoughtbot.com/blog/sass-default it should appear in every variable declaration in theme alpaca